django_project/hw_2/admin.py

```diff
@@ -1,3 +1,65 @@
 from django.contrib import admin
+from django.urls import reverse
+from django.utils.html import format_html
 
-# Register your models here.
+from .models import Client, Product, Order, OrderItem
+
+
+@admin.action(description='Пометить выделенные записи как удаленные')
+def set_deleted(modeladmin, request, queryset):
+    queryset.update(is_deleted=True)
+
+
+class OrderInline(admin.TabularInline):
+    model = Order
+    extra = 0
+
+
+class ClientAdmin(admin.ModelAdmin):
+    list_display = ('name', 'id', 'email', 'phone', 'address', 'created_at', 'is_deleted')
```

Comment:
Кортежи создаются при помощи запятой, а не скобок.
```suggestion
    list_display = 'name', 'id', 'email', 'phone', 'address', 'created_at', 'is_deleted'
```